mxcubeweb/core/components/samplechanger.py

```diff
@@ -183,6 +192,11 @@ def mount_sample_clean_up(self, sample):
 
         sc = HWR.beamline.sample_changer
 
+        try:
+            mount_from_harvester = sc.mount_from_harvester()
```

Comment:
@marcus-oscarsson a review on this file will make the use of mount_from_harvester more clear

Reply:
Ok, thanks I think that some logic in `components/harvester.py` can be moved down to the harvester hardware object